api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto

```diff
@@ -35,4 +35,7 @@ message UdpProxyConfig {
   // The idle timeout for sessions. Idle is defined as no datagrams between received or sent by
   // the session. The default if not specified is 1 minute.
   google.protobuf.Duration idle_timeout = 3;
+
+  // Use the remote downstream ip address as sender ip address when send packets to upstreams.
```

Comment:
Can you clarify this a bit further to explain that packets will then not transit back through Envoy thus performing DSR (direct server return)?

Reply:
Sure. NP :)
I will change that.